$MirOS: ports/devel/cvs/patches/patch-doc_cvs_man_header,v 1.1 2010/09/15 20:56:57 tg Exp $
--- doc/cvs.man.header.orig	Tue Feb  1 22:19:43 2005
+++ doc/cvs.man.header	Thu Sep 16 00:04:39 2010
@@ -1,3 +1,5 @@
+.\" $MirOS: ports/devel/cvs/patches/patch-doc_cvs_man_header,v 1.1 2010/09/15 20:56:57 tg Exp $
+.\"
 .\" This is the man page for CVS.  It is auto-generated from the
 .\" cvs.man.header, cvs.texinfo, & cvs.man.footer files.  Please make changes
 .\" there.  A full copyright & license notice may also be found in cvs.texinfo.
@@ -23,7 +25,9 @@
 .ds Rv \\$3
 .ds Dt \\$4
 ..
-.TH CVS 1 "\*(Dt"
+.de IX
+..
+.TH CVS 1
 .\" Full space in nroff; half space in troff
 .de SP
 .if n .sp
@@ -55,6 +59,7 @@ This manpage is a summary of some of the
 \fBcvs\fP.  It is auto-generated from an appendix of the CVS manual.
 For more in-depth documentation, please consult the
 Cederqvist manual (via the
+cvs(GNU) link in the MirBSD online (HTML) manual pages, the
 .B info CVS
 command or otherwise,
 as described in the SEE ALSO section of this manpage).  Cross-references
